packages/compass-import-export/src/utils/csv.ts

```diff
@@ -84,8 +84,8 @@ const FLOAT = /^\s*-?(\d+\.?|\.\d+|\d+\.\d+)([eE][-+]?\d+)?\s*$/;
 // from papaparse: https://github.com/mholt/PapaParse/blob/aa0046865f1b4e817ebba6966d6baf483e0652d7/papaparse.js#L1025
 const ISO_DATE =
   /^((\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z)))$/;
-const TRUTHY_STRINGS = ['true', 'TRUE', 'True'];
-const FALSY_STRINGS = ['false', 'FALSE', 'False'];
+const TRUTHY_STRINGS = ['t', 'true', 'TRUE', 'True'];
```

Comment:
While manually testing I noticed this pattern for true/false in some open source datasets and figured it would be nice if we could automatically detect and parse it.